src/frontend/src/session.rs

```diff
@@ -785,10 +784,26 @@ impl Session<PgResponseStream> for SessionImpl {
             ));
         }
         let stmt = stmts.swap_remove(0);
-        let rsp = handle(self, stmt, sql, format).await.map_err(|e| {
-            tracing::error!("failed to handle sql:\n{}:\n{}", sql, e);
-            e
-        })?;
+        let rsp = {
+            let handle_fut = handle(self, stmt, sql, format);
+            if cfg!(debug_assertions) {
```

Comment:
Slow query reporting is an important feature even online, so I would suggest to enable logging it with some configuration.